utils/threshAlgos.py

```diff
@@ -472,39 +474,69 @@ def calibrateThrDAC(inputFile,fitRange="0,255",listOfVFATs=None,noLeg=False,outp
         savePlots     - Make a '*.png' file for all plots that will be saved in the output TFile
     """
 
+    if not hasattr(args,"inputFile"):
+        print("No inputFile provided to calibrateThrDac")
+        sys.exit(1)
```

Comment:
it would be more appropriate to provide the exit code [OS_NOINPUT](https://docs.python.org/2/library/os.html#os.EX_NOINPUT) here.

Reply:
done